### PR TITLE
Intergrate React into Rails

### PR DESCRIPTION
**Pull request by mohashyne**
# In  this pull request, i created a new Rails app named 'hello_world-rails_react' :

* Used the command rails new -j webpack my_new_rails_app to initialize the project.
* Verified that the jsbundling-rails gem is added to the Gemfile.
* Initialized the project with Git.
* Installed the necessary node dependencies as detailed in the previous lesson's tutorial.
* Created a babel.config.js file in the root folder of the project.
* Updated webpack.config.js to include the babel loader as per the previous lesson's tutorial.
* Ensured that the project has a Postgres database set up.
* Created a table for storing messages and added 5 different greetings.
* Established an API endpoint that selects a random greeting from the table (implemented a controller with an action and set up a Rails route).
* Created a static view that serves as the root of the app.

### Additional Changes Includes:
* Developed the App component with react-router and integrated it into the static view.
* Designed the Greeting component responsible for displaying a greeting, and configured it as a route within the App component.
* Established a store, an action, and a reducer that connects to the API endpoint for retrieving a random greeting.
* Successfully displayed the random greeting in the Greeting component.
These changes enhance the functionality and user experience of the application, ensuring a dynamic and engaging greeting experience for users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,325 @@
...
+
+** I would like to thank to Microverse for giving this inspiring project **
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (julie-ify) [inline README.md:325]: - [x] Kindly fix your MIT link. It is broken.

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @mohashyne,

Good job so far!

## Highlights!
- Good code organization and working webpack app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mohashyne) [reply README.md:325]: fixed

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: :raising_hand: @mohashyne, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 


## Highlights :green_circle: 

- Two apps successfully connect together to display random greetings :+1: 
- All Links in the readme file are working as expected :+1: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**